src/plugins/nova_plugin_utilities.erl

```diff
@@ -0,0 +1,13 @@
+-module(nova_plugin_utilities).
+-export([parse_options/1]).
+
+parse_options(Options) ->
+    ListOptions = maps:to_list(Options),
+    parse_options(ListOptions, []).
+
+parse_options([], Parsed) -> Parsed;
+parse_options([{K, V}|T], Parsed) when V == true ->
+    parse_options(T, [K|Parsed]);
+parse_options([{K, L}|T], Parsed) when is_list(L) ->
+    Strings = lists:map(fun binary_to_list/1, L),
```

Comment:
Shouldn't this be other way around? List to binary?

L is a list when entering the function.